src/universal/components/NotificationBar/NotificationBar.js

```diff
@@ -33,7 +34,7 @@ const NotificationBar = (props) => {
   return (
     <div className={css(styles.bar)}>
       <div className={css(styles.inlineBlock, styles.message)}>
-        You’ve got a meeting:
+        {`${plural(activeMeetings.length, 'Meeting')} in progress:`}
```

Comment:
Just a note that default function is being called `plural` and not `pluralize` here. Is fine tho!